src/core/renderers/DroppableLayout.tsx

```diff
@@ -99,21 +158,30 @@ interface DropPointProps {
 const DropPoint: React.FC<DropPointProps> = ({ layout, index }) => {
   const classes = useStyles();
   const dispatch = useDispatch();
-  const [{ isOver, schemaElement }, drop] = useDrop({
-    accept: SCHEMA_ELEMENT,
+  const [{ isOver, schemaElement, uiSchemaElement }, drop] = useDrop({
+    accept: [SCHEMA_ELEMENT, UI_SCHEMA_ELEMENT],
     collect: (mon) => ({
       isOver: !!mon.isOver(),
       schemaElement: mon.getItem()?.schemaElement,
+      uiSchemaElement: mon.getItem()?.uiSchemaElement,
     }),
     drop: (): any => {
-      dispatch(Actions.addSchemaElementToLayout(schemaElement, layout, index));
+      if (schemaElement) {
+        dispatch(
+          Actions.addSchemaElementToLayout(schemaElement, layout, index)
```

Comment:
maybe even unify the action and distinguish in the action?
@sdirix  what is your take on this?

Reply:
I would keep the different actions as they are different operations. Dragging in a schema element and dragging in some ui schema element have different consequences.